custom/icds_reports/utils/data_accessor.py

```diff
@@ -48,3 +50,47 @@ def get_program_summary_data_with_retrying(step, domain, config, now, include_te
             sleep(5)
             retry += 1
     return data
+
+
+# keeping cache timeout as 2 hours as this is going to be used
+# in some script/tool which might flood us with requests
```

Comment:
should  I increase the timeout more?

Reply:
I think this is not too bad. We could run the JOIN query once during non-peak load time to see if it's too slow or puts load on the DB and increase the timeout if needed.